src/panels/config/integrations/integration-panels/zwave_js/dialog-zwave_js-reinterview-node.ts

```diff
@@ -0,0 +1,256 @@
+import "@material/mwc-button/mwc-button";
+import { mdiCheckCircle, mdiCloseCircle } from "@mdi/js";
+import {
+  CSSResult,
+  customElement,
+  html,
+  LitElement,
+  property,
+  internalProperty,
+  TemplateResult,
+  css,
+} from "lit-element";
+import "../../../../../components/ha-circular-progress";
+import { createCloseHeading } from "../../../../../components/ha-dialog";
+import { haStyleDialog } from "../../../../../resources/styles";
+import { HomeAssistant } from "../../../../../types";
+import { ZWaveJSReinterviewNodeDialogParams } from "./show-dialog-zwave_js-reinterview-node";
+import { fireEvent } from "../../../../../common/dom/fire_event";
+
+@customElement("dialog-zwave_js-reinterview-node")
+class DialogZWaveJSReinterviewNode extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @internalProperty() private entry_id?: string;
+
+  @internalProperty() private node_id?: number;
+
+  @internalProperty() private _status = "";
+
+  @internalProperty() private _stage?: string;
+
+  private _subscribed?: Promise<() => Promise<void>>;
+
+  public disconnectedCallback(): void {
+    super.disconnectedCallback();
+    this._unsubscribe();
```

Comment:
We should do this in close, as dialogs don't get disconnected

Reply:
Already firing unsubscribe in `closeDialog()` so I removed the entire `disconnectedCallback()`